exercises/concept/captains-log/.docs/instructions.md

```diff
@@ -0,0 +1,42 @@
+## Instructions
+
+Mary is a big fan of the TV series Star Trek: The Next Generation. She often plays pen-and-paper role playing games, where she and her friends pretend to be the crew of the Starship Enterprise. Mary's character is Captain Picard, which means she has to keep the captain's log. She loves the creative part of the game, but doesn't like to generate random data on the spot.
```

Comment:
```suggestion
Mary is a big fan of the TV series Star Trek: The Next Generation.
She often plays pen-and-paper role playing games, where she and her friends pretend to be the crew of the Starship Enterprise.
Mary's character is Captain Picard, which means she has to keep the captain's log.
She loves the creative part of the game, but doesn't like to generate random data on the spot.
```

This follows the Single Sentence Per Line style advocated for in Exercism's markdown files.